docs/archive_data_migration_to_berkeley.md

```diff
@@ -0,0 +1,470 @@
+# Migrating Mainnet Database to Berkeley
+
+Before you start planning the migration of your Mainnet database into the Berkeley version of the Mina network, read this document in entirety.
+
+## Migration apps
+
+Two applications are required to migrate the Mainnet database into the Berkeley version of the Mina network. 
+
+These applications are shipped by MF/O1 Labs:
+
+- **berkeley-migration**
+
+  The berkeley-migration application migrates as much data as possible from the Mainnet database and downloads precomputed blocks to get the window density data. 
+  
+  This application runs against the Mainnet and the new database.
+
+- **replayer app in migration mode**
+
+  The replayer app in migration mode replays the transactions in the partially migrated database and populates the `accounts_accessed` and `accounts_created` tables. This application also performs the checks performed by the standard replayer, except that ledger hashes are not checked because the hard fork ledger has greater depth, which results in different hashes. This application runs only against the new database.
+
+## How to migrate
+
+To get a fully migrated database:
+
+1. First, run the berkeley-migration application.
+2. Next, run replayer app in migration mode. 
+
+The applications are able to work incrementally so that part of the Mainnet database can be migrated and, as new blocks are added to the Mainnet, the new data in the database can be migrated.
+
+To obtain that incrementality, the berkeley-migration application can look at the migrated database and determine the most recent migrated block. It can continue migrating starting at the next block in the Mainnet data. The replayer app in migration mode can use the checkpoint mechanism already in place for the replayer. A checkpoint file indicates the global slot since genesis for starting the replay and the ledger to use for that replay. The application writes new checkpoint files as it proceeds.
+
+To take advantage of the incrementality, you can run a cron job that migrates a day's worth of data at a time (or some other interval). With the cron job in place, at the time of the actual hard fork, only a small amount of data will need to be migrated.
```

Comment:
this information is key! Prep now and migrate only a teensy bit later

(let's add to the intro)